nemo/collections/asr/data/audio_to_text_lhotse_speaker.py

```diff
@@ -0,0 +1,163 @@
+
+    def __getitem__(self, cuts) -> Tuple[torch.Tensor, ...]:
+        spk_targets = [torch.transpose(torch.as_tensor(self.speaker_to_target(c, self.num_speakers, self.num_sample_per_mel_frame, self.num_mel_frame_per_asr_frame, self.spk_tar_all_zero), dtype=torch.float32), 0, 1) for c in cuts]
+        audio, audio_lens, cuts = self.load_audio(cuts)
+        tokens = [torch.as_tensor(self.tokenizer(c.supervisions[0].text, c.supervisions[0].language)) for c in cuts]
+        token_lens = torch.tensor([t.size(0) for t in tokens], dtype=torch.long)
+        tokens = collate_vectors(tokens, padding_value=0)
+        spk_targets = collate_matrices(spk_targets)
+
+
+        return audio, audio_lens, tokens, token_lens, spk_targets
+    
+
+    def speaker_to_target(self, cut, num_speakers: int = 4, num_sample_per_mel_frame: int = 160, num_mel_frame_per_asr_frame: int = 8, spk_tar_all_zero: bool = False):
+        '''
+        get rttm samples corresponding to one cut, generate speaker_audio_mask numpy.ndarray
+        '''
+
+        #get cut-related segments from rttms
+        segments_iterator = self.rttms.find(recording_id=cut.recording_id, start_after=cut.start, end_before=cut.end, adjust_offset=True)
+
+        segments = [s for s in segments_iterator]
+
+        #generate speaker id according to arrival time
+        segments.sort(key = lambda rttm_sup: rttm_sup.start)
+
+        seen = set()
+        seen_add = seen.add
+        speaker_ats = [s.speaker for s in segments if not (s.speaker in seen or seen_add(s.speaker))]
+        
+        speaker_to_idx_map = {
+                spk: idx
+                for idx, spk in enumerate(speaker_ats)
+        }
+
+        
+        #initialize mask matrices (num_speaker, encoder_hidden)
+        mask = np.zeros((num_speakers, int(np.ceil(np.ceil((cut.num_samples + 1) / num_sample_per_mel_frame) / num_mel_frame_per_asr_frame))))
+
+        if not spk_tar_all_zero:
+            for rttm_sup in segments:
+                speaker_idx = speaker_to_idx_map[rttm_sup.speaker]
+                #only consider the first <num_speakers> speakers
+                if speaker_idx < 4:
+                    st = (
+                                compute_num_samples(rttm_sup.start, cut.sampling_rate)
+                                if rttm_sup.start > 0
+                                else 0
+                            )
+                    et = (
+                                compute_num_samples(rttm_sup.end, cut.sampling_rate)
+                                if rttm_sup.end < cut.duration
+                                else compute_num_samples(rttm_sup.duration, cut.sampling_rate)
+                            )                   
+                    mask[speaker_idx, int(np.ceil(np.ceil((st + 1) / num_sample_per_mel_frame) / num_mel_frame_per_asr_frame)):int(np.ceil(np.ceil((et + 1) / num_sample_per_mel_frame) / num_mel_frame_per_asr_frame))] = 1
+
+        return mask
+
+
+class TokenizerWrapper:
```

Comment:
What is the need for this `TokenizerWrapper` to be re-defined?
If there is no change from the original one, please import from the original.